.stylelintrc

```diff
@@ -0,0 +1,59 @@
+{
+  "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
+
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-name-case": "lower",
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "number-zero-length-no-unit": true,
+
+    "string-no-newline": true,
+    "string-quotes": "single",
+
+    "unit-case": "lower",
+    "unit-no-unknown": true,
+    "unit-whitelist": ["px", "%", "deg", "ms", "em"],
+
+    "value-list-comma-space-after": "always-single-line",
+    "value-list-comma-space-before": "never",
+
+    "shorthand-property-no-redundant-values": true,
```

Comment:
Bad margin: 0 0 5px 0; Good margin: 0 0 5px;
Always use shorthand.  Rule will give you shorthand that it expects.